<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Nibble Entropy GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <property name="spacing">
     <number>8</number>
    </property>
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>

    <!-- Таблица 16×16 -->
    <item>
     <widget class="QTableView" name="tableView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>

    <!-- Панель метрик -->
    <item>
     <widget class="QWidget" name="metricsWidget">
      <layout class="QHBoxLayout" name="metricsLayout">
       <property name="spacing">
        <number>16</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>

       <!-- N -->
       <item>
        <widget class="QLabel" name="labelNTitle">
         <property name="text">
          <string>&lt;b&gt;N:&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblN">
         <property name="text">
          <string>–</string>
         </property>
        </widget>
       </item>

       <!-- H тек -->
       <item>
        <widget class="QLabel" name="labelHTitle">
         <property name="text">
          <string>&lt;b&gt;H тек:&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblH">
         <property name="text">
          <string>–</string>
         </property>
        </widget>
       </item>

       <!-- H макс -->
       <item>
        <widget class="QLabel" name="labelHmaxTitle">
         <property name="text">
          <string>&lt;b&gt;H макс:&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblHmax">
         <property name="text">
          <string>–</string>
         </property>
        </widget>
       </item>

       <!-- H отн -->
       <item>
        <widget class="QLabel" name="labelHrefTitle">
         <property name="text">
          <string>&lt;b&gt;H отн:&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblHref">
         <property name="text">
          <string>–</string>
         </property>
        </widget>
       </item>

       <!-- Спейсер -->
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>

      </layout>
     </widget>
    </item>
   </layout>
  </widget>

  <!-- Меню -->
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect><x>0</x><y>0</y><width>800</width><height>22</height></rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>

  <!-- Тулбар -->
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Файл</string>
   </property>
   <addaction name="actionOpen"/>
  </widget>

  <!-- Статусбар -->
  <widget class="QStatusBar" name="statusbar"/>

  <!-- Экшены -->
  <action name="actionOpen">
   <property name="text">
    <string>Открыть…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>

 </widget>
 <resources/>
 <connections/>
</ui>
